include/net/nrf_cloud.h

```diff
@@ -523,6 +524,11 @@ struct nrf_cloud_init_param {
 	 * Only used if CONFIG_NRF_CLOUD_FOTA_FULL_MODEM_UPDATE is enabled.
 	 */
 	struct dfu_target_fmfu_fdev *fmfu_dev_inf;
+	/** Similarly than with nrf_cloud_os_mem_hooks_init(),
```

Comment:
```suggestion
	/** Optional hooks to override memory management functions.
	 *  If set, @ref nrf_cloud_os_mem_hooks_init will be called by @ref nrf_cloud_init
	 */
```

*not sure if the `@ref` works for items defined outside of this header.

Reply:
Changed